server/cloudformation/ecs.go

```diff
@@ -117,7 +118,17 @@ func (p *ECSServiceResource) create(properties *ECSServiceProperties) (string, e
 		return "", fmt.Errorf("error creating service: %v", err)
 	}
 
-	return *resp.Service.ServiceArn, nil
+	arn := resp.Service.ServiceArn
+	if err := p.ecs.WaitUntilServicesStable(&ecs.DescribeServicesInput{
```

Comment:
Turns out, the waiter will check up to 40 times, with a 15 second delay between checks, so this will wait up to 10 minutes, which seems about right.

https://github.com/aws/aws-sdk-go/blob/2e1a49c71fb7d8f6b7e8cca2259ed9b68d9b97ce/service/ecs/waiters.go#L41-L42